kuma/contributions/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+lang_urlpatterns = [
+    url(r'^confirmation/success/?$',
```

Comment:
I think a confirmation screen used to be part of the process, but it isn't anymore. The URL should simplify from ``/contribute/confirmation/success`` to ``/contribute/success``.